packages/opc-footer/README.md

```diff
@@ -0,0 +1,99 @@
+# This is opc-footer 👋
+
+Web component based on lit-html
```

Comment:
Add some description for the Footer component.

And also some shields for version, build and maintenance status.